heartbeat/nfsserver

```diff
@@ -854,6 +961,8 @@ nfsserver_stop ()
 		ocf_log info "NFS server stopped"
 	fi
 
+	case $EXEC_MODE in [23]) remove_unshare_uts_dropins; esac
```

Comment:
Ditto.

Reply:
ditto ;-) not necessary, but I'll add it for style.